### PR TITLE
Kan 6 harsha payment

### PR DESCRIPTION
**Pull request by Harsha1varthan**
#Added Every Changes according to the Request

### CODE REVIEW

File: Frontend/src/States/state.js
Comment (aravind-digitech) [inline Frontend/src/States/state.js:1]: pls move this file to the particular component's folder

File: Frontend/src/components/CheckOut/constants/constant.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const LandLine = 91
```
Comment (aravind-digitech) [inline Frontend/src/components/CheckOut/constants/constant.js]: use CAPS for names eg LANDLINE

File: Frontend/src/css/Input.module.css
Comment (aravind-digitech) [inline Frontend/src/css/Input.module.css:1]: revert changes in all files under css folder

File: Frontend/src/App.css
Comment (aravind-digitech) [inline Frontend/src/App.css:1]: pls move these changes to index.css file and also follow naming convention like checkOutMain-least-padding

File: Frontend/src/index.css [hunk 2]
```diff
@@ -12,18 +12,18 @@
   --font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
   --font-normal: 400;
   --font-bold: 700;
-  --base-font-size: 16px;
+  --base-font-size: 20px;
```
Comment (aravind-digitech) [inline Frontend/src/index.css]: pls avoid changes to existing props, pls create one with component prefix if required eg innerbox-base-font-size

**Review by aravind-digitech — CHANGES_REQUESTED** [review]

**Comment by aravind-digitech** [comment]
@Murali03M @Harsha1varthan @GREEDnim pls review and approve

**Review by aravind-digitech — APPROVED** [review]

**Review by Sathish111j — APPROVED** [review]

**Review by aravind-digitech — APPROVED** [review]

**Review by jayakanthansubramani — APPROVED** [review]

**Pull request merged**